client/web/src/nav/StatusMessagesNavItem.scss

```
@@ -7,6 +7,7 @@
         min-width: 22rem;
         max-width: 22rem;
         padding: 1rem 1rem 0 1rem;
+        user-select: text;
```

Any reason to not allow `all`? (feel free to merge straight away regardless of the answer)
```suggestion
        user-select: all;
```

---

I think, selection is a bit more natural. I'll merge it but we can always revisit. Thank you, Felix!